### DISCUSSION TITLE
Duda organización del curso

### DISCUSSION THREAD

**Discussion by PatricioNovellinoLaunchx** [Launch-X-Latam/MisionFrontEnd#270] [Q&A]
Hola que tal a todos. Últimamente estuve muy ocupado por asuntos de facultad y trabajo pero ya voy a poder meterle mas a las practicas. En teoría no estoy tan atrasado voy por la primer clase de CSS.

Hay algo que tenga fecha de entrega? algún formulario donde empecemos a subir las practicas? O todo se mantiene en github aun?

Gracias a todos de antemano

**Comment by AlvaroFierro** [comment]
Hola @PatricioNovellino 

Como tal no tiene fecha de entrega, solamente si procura no atrasarte, porque después vamos a rellenar un formulario en el que debemos de poner el link a nuestro repositorio y avanzar a la segunda etapa del curso, que es lo de Backend.

Pero puedes ver que algunas practicas dicen que la fecha de entrega es el 4 de noviembre, pero es para pasar a la segunda etapa
Por lo que entiendo, solamente los que hagan todos los ejercicios, pueden pasar a la segunda etapa
